### ISSUE TITLE
GitHub actions

### ISSUE THREAD

**Issue by hoshsadiq** [maxatome/install-go#8]
I'd like to raise a PR to enable GitHub actions, but before I do that, I wanted to check if you'd be okay with that. It would require some admin work on your side to publish it on the market. Would this be okay?

**Comment by maxatome** [comment]
What is the needed "admin work"? Is it one shot?

**Comment by hoshsadiq** [comment]
Yes, basically. It's just enabling it in the settings of the repo. See [here](https://docs.github.com/en/actions/creating-actions/publishing-actions-in-github-marketplace) for the documentation.

**Comment by maxatome** [comment]
Fine, if needed I can set you as collaborator.

**Comment by hoshsadiq** [comment]
Cool, thanks. I'll develop the action, then will raise a PR. We can take it from there!
